client/dashboard/summary-button/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+	Card,
+	CardBody,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Button,
+	Icon,
+} from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import CoreBadge from 'calypso/components/core/badge';
+import { SummaryButtonProps, SummaryButtonFieldProps } from './types';
+import './style.scss';
+
+const noop = () => {};
+
+function SummaryButton(
+	{
+		title,
+		to,
+		decoration,
+		description,
+		strapline,
+		fields,
+		leadsToNestedPage = true,
+		onClick = noop,
+		disabled,
+		density = 'low',
+	}: SummaryButtonProps,
+	ref: React.ForwardedRef< HTMLAnchorElement >
+) {
+	const hasLowDensity = density === 'low';
+	const badges = fields?.length && (
+		<HStack spacing={ 1 } justify="flex-start" style={ { minWidth: 'fit-content' } }>
+			{ fields?.map( ( field: SummaryButtonFieldProps ) => (
+				<CoreBadge key={ field.text } intent={ field.intent || 'default' }>
+					{ field.text }
+				</CoreBadge>
+			) ) }
+		</HStack>
+	);
+	return (
+		<Button
+			ref={ ref }
+			href={ to }
```

Comment:
to is a route or an href?

Reply:
It should be named "href" for `SummaryButton` then because `SummaryButton` is just a regular button with some design choices, it has nothing to do with the router.